PowerEditor/src/MISC/PluginsManager/Notepad_plus_msgs.h

```diff
@@ -454,6 +456,32 @@ enum Platform { PF_UNKNOWN, PF_X86, PF_X64, PF_IA64, PF_ARM64 };
 		HICON	hToolbarIconDarkMode;
 	};
 
+	#define NPPM_GETEXTERNALLANGAUTOINDENTMODE  (NPPMSG + 103)
+	// BOOL NPPM_GETEXTERNALLANGAUTOINDENTMODE(const TCHAR *languageName, LangAutoIndentMode &autoIndentMode)
+	// Get LangAutoIndentMode for an installed external programming language.
+	// - Standard means Notepad++ will keep the same TAB indentation between lines;
+	// - Extended means Notepad++ will perform a C-Language style indentation for the selected external language;
+	// - Custom means a Plugin will be controlling auto-indentation for the current language.
+	// returned values: TRUE for successful searches, otherwise FALSE.
+
+	#define NPPM_SETEXTERNALLANGAUTOINDENTMODE  (NPPMSG + 104)
+	// BOOL NPPM_SETEXTERNALLANGAUTOINDENTMODE(const TCHAR *languageName, LangAutoIndentMode autoIndentMode)
+	// Set LangAutoIndentMode for an installed external programming language.
+	// - Standard means Notepad++ will keep the same TAB indentation between lines;
+	// - Extended means Notepad++ will perform a C-Language style indentation for the selected external language;
+	// - Custom means a Plugin will be controlling auto-indentation for the current language.
+	// returned value: TRUE if function call was successful, otherwise FALSE.
+
+	#define NPPM_ISAUTOINDENTATIONON  (NPPMSG + 105)
+	// BOOL NPPM_ISAUTOINDENTATIONON(0, 0)
+	// Returns the current Use Auto-Indentation setting in Notepad++ Preferences.

```

Comment:
Please rename the above to following:
* `NPPM_GETEXTERNALLEXERAUTOINDENTMODE`
* `NPPM_SETEXTERNALLEXERAUTOINDENTMODE`
* `NPPM_ISAUTOINDENTON`

Sorry for changing the naming again.